plugins/modules/luks_device.py

```diff
@@ -217,6 +217,47 @@
             - "Will only be used on container creation."
         type: int
         version_added: '1.5.0'
+    perf_same_cpu_crypt:
+        description:
+            - "This option allows user to perform encryption using the same cpu that IO was submitted on."
+            - "The default is to use an unbound workqueue so that encryption work is automatically balanced between available CPUs."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    perf_submit_from_crypt_cpus:
+        description:
+            - "This option allows user to disable offloading writes to a separate thread after encryption."
```

Comment:
```suggestion
            - "Allows the user to disable offloading writes to a separate thread after encryption."
```
Please apply similar transformations to the other options.